src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -140,14 +140,14 @@ public function load(array $configs, ContainerBuilder $container)
         $this->registerAnnotationsConfiguration($config['annotations'], $container, $loader);
         $this->registerPropertyAccessConfiguration($config['property_access'], $container);
 
-        if ($this->isConfigEnabled($container, $config['serializer'])) {
-            $this->registerSerializerConfiguration($config['serializer'], $container, $loader);
-        }
-
         if ($this->isConfigEnabled($container, $config['property_info'])) {
             $this->registerPropertyInfoConfiguration($config['property_info'], $container, $loader);
         }
 
+        if ($this->isConfigEnabled($container, $config['serializer'])) {
```

Comment:
I check below if the service `property_info` exists (have you a better idea?). this assumes that property info is built before serializer.